### PR TITLE
More stable default configuration

### PR DESCRIPTION
**Pull request by BLaunet**
Here a suggestion of improvements to the default configuration.
- We try to use as much as possible the path defined in the config file, to avoid duplicates.
- Warnings are raised when no path has been provided for GALFIT or SDSS PSF tool binaries

### CODE REVIEW

File: roou.py [hunk 1]
```diff
@@ -26,16 +21,17 @@
...
 # AGN point sources.
 path_to_Hubble_psf = '/mnt/ds3lab/dostark/hubble_psf_many_stars.fits'
 #path_to_Hubble_psf = '/mnt/ds3lab/dostark/PSFSTD_WFC3IR_F160W.fits'
-tmp_GALFIT = '/mnt/ds3lab/dostark/hubble_z_'+str(conf.redshift)+\
-             '/'+conf.filter_+'-band/tmp_for_GALFIT_'+str(conf.ext)+\
-             '_'+str(conf.stretch_type)+'_'+str(conf.scale_factor)+'/'
+
+# Directories for temporary files
+tmpdir_for_SExtractor = conf.stretch_setup+'/tmp_for_SExtractor/'
+tmp_GALFIT = conf.stretch_setup+'/tmp_for_GALFIT/'
```
Comment (BLaunet) [inline roou.py]: @dominicstark Is it a problem to change the paths of these temporary directories like that ? Information is the same, just organized in a different way

**Comment by BLaunet** [comment]
Left to correct:

- in photometry.get_stars_from_field, some absolute paths are hardcoded (FILTER_NAME, CHECKIMAGE_NAME, STARNNW_NAME)

- in general, what refers to Hubble PSF is hardcoded

File: photometry.py [hunk 2]
```diff
@@ -17,6 +17,13 @@
 sex = 'sextractor'
 fields_core_path = conf.core_path+'/source/sdss/dr12/plates/'
 
+# Directories containing default files for SExtractor:
+SExtractor_default_files_path = ''
```
Comment (BLaunet) [inline photometry.py]: Great ! :+1:
Now the best would be to provide a default folder with these files, and put its path here

Comment (dominicstark) [reply photometry.py]: Hm you are right. So lets put them into the tarball. and add the path here.

Comment (BLaunet) [reply photometry.py]: It's just a few Kbytes right ? No need for a tarball, just include the folder in the repository

Comment (dominicstark) [reply photometry.py]: Ok!

**Pull request merged**